packages/react-instantsearch/src/widgets/Breadcrumb.js

```diff
@@ -82,4 +87,16 @@ import BreadcrumbComponent from '../components/Breadcrumb.js';
  *   );
  * }
  */
-export default connectBreadcrumb(BreadcrumbComponent);
+
+const Widget = props => (
```

Comment:
We should destruct the `props` in order to extract `header`, `footer` to avoid to pass them down the three. We should also add `PropTypes` on the `Widget` component.